he/psi_he.ts

```diff
@@ -9139,7 +9139,7 @@ This may mean you are already running another instance of Psi. You may experienc
     <message>
         <location filename="../psi/src/psicontactlistviewdelegate.cpp" line="149"/>
         <source>%1 (%2)</source>
-        <translation type="unfinished"></translation>
+        <translation></translation>
```

Comment:
Maybe we should add RLM character for proper alignment.

%1 RLM(%2)
RLM%1 RLM(%2)
RLM%1 RLM(%2)RLM

What do you think?